packages/core/README.md

```diff
@@ -0,0 +1,76 @@
+# @secbox/core
+
+The base package can be used to obtaining config including credentials from different sources, and provide simplified abstraction to handle integration events and commands.
+
+## Setup
+
+```bash
+npm i -s @secbox/core
+```
+
+## Usage
+
+### Configuration
+
+First, you need to generate a new instance of `Configuration`.
+
+```ts
+import { Configuration } from '@secbox/core';
+
+const config = new Configuration({
+    api: 'app.neuralegion.com',
+    bus: 'EventBus'
+    credentials: {
+      token: 'xxxxxxx.xxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
+    }
+  });
+```
+
+ After that, you can inject it using `container`.
+
+```ts
+const injectedConfig = config.container.resolve(Configuration);
+```
+
+#### Options
+
+```ts
+interface ConfigurationOptions {
+  bus?: string;
+  api?: string;
+  credentials?: Credentials;
+  credentialProviders?: Array<CredentialProvider>;
+}
+```
+
+`bus` - key of exchange\
+`api` - URL that will be used to connect to the queue\
+`credentials` -  credentials that are needed to get access to the queue\
+`credentialProviders` - array of providers that provide credentials
+
+#### Credentials
+
+You have two ways to pass credentials:
+  - `credentials` property
```

Comment:
Use `option` instead of `property`, since we are speaking about configuration, and not about objects.

```suggestion
  - `credentials` option
```